isis/src/control/objs/BundleUtilities/BundleMeasure.cpp

```diff
@@ -108,6 +108,45 @@ namespace Isis {
   }
 
 
+  /**
+   * Sets the BundleMeasure's image partial
+   *
+   * @param imagePartial BundleMeasure's imagePartial matrix
+   */
+  void BundleMeasure::setImagePartial(const LinearAlgebra::Matrix &imagePartial) {
+    *m_imagePartial = &imagePartial;
```

Comment:
`imagePartial` is a reference so you don't want the `&` here. Instead what we want to do is make a copy of imagePartials and then save off the pointer to it:

```
// Delete any old partial matrix we have
if (m_imagePartial) {
  delete m_imagePartial;
  m_imagePartial = nullptr;
}

// Save a copy of the new one
m_imagePartial = new LinearAlgebra::Matrix(imagePartial);
```

Reply:
What the current implementation is doing, is saving off a pointer to the const reference to the imagePartials. That does not make a deep copy and will point to deleted memory after imagePartials is deleted in the bundle adjust.